app/javascript/components/background.js

```diff
@@ -0,0 +1,12 @@
+const changeColor = () => {
+
```

Comment:
const banner = document.querySelector(".banner")

Reply:
const banner?

Reply:
changed

Reply:
Bad changes ... that's why it doesn't work
`const changeColor = document.querySelector(".banner") => {`
should be
```
const changeColor = () => {
const banner = document.querySelector(".banner") 
```